e2e/footer.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  page.goto('http://localhost:3000/#footer');
+});
+
+test.describe('Test Footer Navigation Links', () => {
+  test('About us link should navigate to about us section', async ({
+    page,
+  }) => {
+    await page.getByRole('link', { name: 'About us' }).click();
+    await expect(page).toHaveURL('/#about-us');
```

Comment:
There is a chance that these footer tests become flaky in the future or are testing the wrong links since this selector would also match the links in the header. I suggest we scope the selectors here to the footer and also adjust the header tests and scope those ones to the header instead of selecting them out of the entire page. Here is an example. Do this also for the events and contact us tests

```tsx
test('About us link should navigate to about us section', async ({
  page,
}) => {
  const footerLocator = page.locator('footer');
  await footerLocator.getByRole('link', { name: 'About us' }).click();
  await expect(page).toHaveURL('/#about-us');
});
```

```tsx
test('About us link should navigate to about us section', async ({
  page,
}) => {
  const headerLocator = page.locator('header');
  await headerLocator.getByRole('link', { name: 'About us' }).click();
  await expect(page).toHaveURL('/#about-us');
});
```

Reply:
Good suggestions. Implemented via [fd3f251](https://github.com/reactdeveloperske/reactdevske-website/pull/107/commits/fd3f2514fd2ff1fb5bbaa650f71d77d0777ca95a)